proxy/api/src/identity.rs

```diff
@@ -95,11 +97,18 @@ impl From<Ethereum> for EthereumClaimExtV1 {
 /// # Errors
 pub async fn create(
     peer: &coco::net::peer::Peer<BoxedSigner>,
-    metadata: &Metadata,
+    metadata: Metadata,
 ) -> Result<Identity, error::Error> {
-    let handle = metadata.handle.clone();
-    let ethereum = metadata.ethereum.clone().map(Into::into);
-    let user = coco::state::init_owner(peer, handle, ethereum).await?;
+    let person = identities_payload::Person {
+        name: metadata.handle.into(),
+    };
+    let mut payload = identities_payload::PersonPayload::new(person);
+    if let Some(ethereum) = metadata.ethereum {
+        payload
+            .set_ext(EthereumClaimExtV1::from(ethereum))
```

Comment:
Ahhh I didn't realise `set_ext` returned an error. My thinking for `impl Into<PersonPayload>` in `state.rs` was so that `Metadata` could implement `From` and then you'd just pass it into `init_owner`. I'm wondering if that's an improvement now :thinking: 

I think a `TryFrom` for `Metadata` might be cleaner here though, what do you think?

Reply:
> `TryFrom` for `Metadata`

Why haven't I thought about it :facepalm:

Reply:
Fixed, now it turned out that we don't need any extra test tools, the production API is IMO good enough